llabake/Hello-Books
# Ft manage signup state using redux 153937223

#### What does this PR do?
This PR implements sign up using redux to manage state
#### Description of Task to be completed?
User should be able to signup on the application
#### How should this be manually tested?
* clone the repo
* cd into the project directory
* Run npm install to install dependencies
* Run npm run dev to start the client server
* Run npm run start-dev to start the server-side
* Visit the port where the webpack-dev-server started, navigate to the signup page
* Fill the sign up form and click register, 
  * If sign up is successful, a success message is displayed and the user is redirected to the home page
  * If sign up is unsuccessful, an error message is displayed.
#### What are the relevant pivotal tracker stories?
153937223
#### Any background context you want to add?
N/A
#### Screenshots
![image](https://user-images.githubusercontent.com/29184825/34456064-10d10a72-ed8e-11e7-9efb-fda401cde9c0.png)
